inc/spbc-backups.php

```diff
@@ -1,26 +1,28 @@
 <?php
 
 use CleantalkSP\SpbctWP\Helper as SpbcHelper;
```

Comment:
Redundant?

Reply:
Thanks) You are correct. Fixed: https://github.com/CleanTalk/security-malware-firewall/commit/ae504054341e358da7ba86715b65c0bb099ec59c